test/functional/wallet_multiwallet.py

```diff
@@ -304,9 +304,9 @@ def wallet_file(name):
         # Fail to load duplicate wallets
         path = os.path.join(self.options.tmpdir, "node0", "regtest", "wallets", "w1", "wallet.dat")
         if self.options.descriptors:
-            assert_raises_rpc_error(-4, f"Wallet file verification failed. SQLiteDatabase: Unable to obtain an exclusive lock on the database, is it being used by another instance of {self.config['environment']['PACKAGE_NAME']}?", self.nodes[0].loadwallet, wallet_names[0])
+            assert_raises_rpc_error(-35, f"Wallet file verification failed. Wallet \"{wallet_names[0]}\" is already loaded.", self.nodes[0].loadwallet, wallet_names[0])
```

Comment:
In commit "rpc: Improve error when wallet is already loaded" (ca8eba38a19a3c1eac8fc41ae7f9f9a17660710c)

Note for other reviewers: it seemed like a potential concern that this test is dropping coverage for the "Unable to obtain an exclusive lock" error, which can still be triggered. But it looks like there is another check for that error in  in `feature_filelock.py`, so this should be ok.